### PR TITLE
Minor UX update and version bumps

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#6378

*Description of changes:*

* Minor style update
* Component and App version updates

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "0.5.40",
+  "version": "0.6.00",
```
Comment (severinbeauvais) [inline package.json:3]: Double zero, wow!

Comment (cameron-freshworks) [reply package.json:3]: hahah oops

Comment (cameron-freshworks) [reply package.json:3]: Double `00` makes "cents" doesn't it?

😆 Good thing no one will see that joke..

**Review by severinbeauvais — APPROVED** [review]
L00ks good.

File: package.json (hunk 1 above)
Comment (severinbeauvais) [reply package.json:3]: Augh.

**Comment by severinbeauvais** [comment]
At this rate we will go to "1" soon enough, so fix it or not, it probably doesn't matter.

**Pull request merged**